kinesis/src/main/scala/akka/stream/alpakka/kinesis/CommittableRecord.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.kinesis
+
+import akka.Done
+import software.amazon.kinesis.lifecycle.ShutdownReason
+import software.amazon.kinesis.retrieval.KinesisClientRecord
+import software.amazon.kinesis.retrieval.kpl.ExtendedSequenceNumber
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class CommittableRecord(
```

Comment:
This is a stream element type, right? Would factory methods simplify this API?

Reply:
Not sure about this comment. This class it's only instantiated from the `impl` code. Is it ok if I just make the constructor private?

Reply:
Yes, make it `@InternalApi private[kinesis]`. You may want to consider unit testing support like in [the reference connector](https://github.com/akka/alpakka/blob/master/reference/src/main/scala/akka/stream/alpakka/reference/testkit/MessageFactory.scala).